src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -165,6 +205,7 @@ CHIP_ERROR OnTransferSessionBegin(TransferSession::OutputEvent & event)
 
                     if (!mInitialized || mTransferGeneration != transferGeneration) {
                         // Callback for a stale transfer.
+                        LogErrorOnFailure(mTransfer.AbortTransfer(bdx::StatusCode::kUnexpectedMessage));
```

Comment:
This does not make sense to me.  If `mTransferGeneration != transferGeneration` that means that this callback is for a differrent transfer attempt than the one `mTransfer` corresponds to.  Why are we aborting `mTransfer` then?  It's for a different attempt.

Reply:
This code i forgot to remove. This is not even being hit.

Reply:
done